requires.py

```diff
@@ -1,31 +1,94 @@
 from charms.reactive import (
     toggle_flag,
-    ResponderEndpoint,
+    Endpoint,
 )
 
-from .common import JobRequest
+import json
+from copy import deepcopy
+from uuid import uuid4
 
+class Job(object):
 
-class PrometheusManualRequires(ResponderEndpoint):
-    REQUEST_CLASS = JobRequest
+    def __init__(self, dic):
+        for key in dic:
+            setattr(self, key, dic[key])
+
+    # seyeong kim : I moved this code from request_reponse pattern as
+    # compatibility
+    def to_json(self, ca_file=None, cert_file=None, key_file=None):
```

Comment:
I don't know the details of what the content needs to be, so I haven't looked over this for correctness at all.

Reply:
I tried to match the same format as previous(when using request-response pattern) one. so just copied this function from that pattern.